### PR TITLE
Cast and crew

### PR DESCRIPTION
**Pull request by mariuszmmm**
The sections "Cast" and "Crew" were added to movieDetails.
The sections "Movies - cast" and "Movies - crew" were added to peopleDetails.

### CODE REVIEW

File: src/components/MovieTile/styled.js [hunk 1]
```diff
@@ -2,13 +2,16 @@ import { NavLink } from "react-router-dom/cjs/react-router-dom.min";
...
+  flex-direction: column;
+  cursor: pointer;
   padding: 16px;
   border-radius: 5px;
   background-color: ${({ theme }) => theme.color.white};
   box-shadow: 0px 4px 12px 0px ${({ theme }) => theme.color.boxShadow};
   transition: 0.4s;
   width: 100%;
-  overflow: hidden;
+  gap: 16px;
```
Comment (Urbork) [inline src/components/MovieTile/styled.js]: i think this does not work, there is no gap in tiles

**Review by Urbork — APPROVED** [review]
Works very well, nice job

File: src/components/MovieTile/styled.js (hunk 1 above)
Comment (mariuszmmm) [reply src/components/MovieTile/styled.js]: I didn't notice that. The issue arose after merging with the main branch, where additional links appeared and disrupted the layout. It's fixed now.

File: src/components/PeopleList/PeopleTile/styled.js [hunk 2]
```diff
@@ -1,36 +1,44 @@
...
     padding: 8px;
+    gap: 8px;
     min-height: 245px;
   }
 `;
 
 export const Image = styled.img`
   margin: 0;
-  margin-bottom: clamp(8px, 1vw, 12px);
-  width: 100%;
```
Comment (Adrode) [inline src/components/PeopleList/PeopleTile/styled.js:33]: I think that by removing these lines of code, the photo inside the PeopleTile does not change its size, which is why the layout breaks on smaller screens.
cast-crew-images

**Review by Adrode — CHANGES_REQUESTED** [review]
Nice job, appreciate adding credits to details and a lot of smaller changes.
Please fix photos shrinking on smaller screens.

**Pull request merged**